rust-runtime/aws-smithy-async/Cargo.toml

```diff
@@ -31,3 +31,6 @@ all-features = true
 targets = ["x86_64-unknown-linux-gnu"]
 rustdoc-args = ["--cfg", "docsrs"]
 # End of docs.rs metadata
+
+[package.metadata.release-tooling]
```

Comment:
we should probably call this `smithy-rs-release-tooling`

Reply:
Addessed in 913116e.